### PR TITLE
deps: upgrade dependencies to patch security warnings

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

This PR will patch all the [security warnings](https://github.com/mindvalley/wukong-cli/security/dependabot) by upgrading our crates.

Ticket:

### Fixed
- [x] upgrade `chrono` to the latest version to fix [Segmentation fault in time](https://github.com/mindvalley/wukong-cli/security/dependabot/21) (https://github.com/chronotope/chrono/releases/tag/v0.4.30)
- [x] upgrade `criterion` to fix [atty potential unaligned read ](https://github.com/mindvalley/wukong-cli/security/dependabot/23)
- [x] remove `async-tungstenite` to fix [Tungstenite allows remote attackers to cause a denial of service](https://github.com/mindvalley/wukong-cli/security/dependabot/29)
- [x] upgrade `git2` to fix [libgit2-sys affected by memory corruption, denial of service, and arbitrary code execution in libgit2](https://github.com/mindvalley/wukong-cli/security/dependabot/41)

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/207#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cli/src/output/table.rs
Comment (mfauzaan) [inline cli/src/output/table.rs:60]: Anyway to avoid unwrap?

**Review by mfauzaan — CHANGES_REQUESTED** [review]

File: cli/src/output/table.rs
Comment (jk-gan) [reply cli/src/output/table.rs:60]: Ya we could, but this is just for overflow checking, so i think instead of using a default value, just make it panic. So we know the overflow happen.

Comment (jk-gan) [reply cli/src/output/table.rs:60]: because overflow is very hard to notice

Comment (mfauzaan) [reply cli/src/output/table.rs:60]: Cool

**Review by mfauzaan — APPROVED** [review]
LGTM!

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/207#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/207#pullrequestreview-1933529773" title="Approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
